test/integration/functional_test.go

```diff
@@ -568,6 +572,40 @@ func validateMySQL(ctx context.Context, t *testing.T, profile string) {
 	}
 }
 
+// validateFileSync start up VM and test sync file between host and VM
+func validateFileSync(ctx context.Context, t *testing.T, profile string) {
+	// This sync command will instruct the minikube to start so that it
+	// will trigger the sync process. The minikube HAS already been
+	// started in the serial test above.
+	startArgs := append([]string{"start", "-p", profile, "--alsologtostderr", "-v=3"}, StartArgs()...)
+
+	// 1. copy random file to MINIKUBE_HOME/files/etc
+	tempfile := filepath.Join(localpath.MiniPath(), "/files/etc/filesync.txt")
+	err := copy.Copy("./testdata/filesync.txt", tempfile)
+	if err != nil {
+		t.Fatalf("copy: %v", err)
+	}
+
+	// 2. start VM (to trigger the file sync process)
+	c := exec.CommandContext(ctx, Target(), startArgs...)
+	rr, err := Run(t, c)
+	if err != nil {
+		t.Errorf("%s failed: %v", rr.Args, err)
+	}
+
+	// 3. check file existence
+	statusCmd := "(ls /etc/filesync.txt  >> /dev/null 2>&1 && echo yes) || echo no"
```

Comment:
No need to make the check so complicated, just use `cat /etc/filesync.txt`and check for the error code.

If you want to make the check more comprehensive, assert that the output matches `testdata/filesync.txt`.